### ISSUE TITLE
Commits with comma in the Subject aren't parsed correctly

### ISSUE THREAD

**Issue by jamesbroadhead** [seymen/git-last-commit#10]
The use of ',' as the splitCharacter means that valid input which contains commas breaks parsing. 

Instead, a deep-unicode character could be used

eg. 
```
JIRA-43543 I did this, and that. (#21483)
``` 
is parsed as 
```
JIRA-43543 I did this
```

**Comment by seymen** [comment]
Instead of us changing the default split character, could you use `splitChar` parameter as explained in the readme to solve this?

**Comment by jamesbroadhead** [comment]
That's a fair response - thank you! 

That said, I think that there are better characters out there which could be used as the default.

**Comment by jamesbroadhead** [comment]
@seymen I tried this, but without success - I think it's got to do with the evaluation order of the default splitCharacter and the one passed in by argument

**Comment by seymen** [comment]
oh, seems like a bug then. Let me have a look at this.

**Comment by seymen** [comment]
Fixed by #11
